app/src/vuex/modules/keystore.js

```diff
@@ -57,6 +57,32 @@ export default () => {
       const { getSeed } = await import("@lunie/cosmos-keys")
       return getSeed()
     },
+    async testSeed(store, { networkId, address, seedPhrase }) {
+      const networkObject = store.getters.networks.find(
+        ({ id }) => id === networkId
+      )
+      const walletVariations = JSON.parse(networkObject.HDPaths).reduce(
+        (all, HDPath) => {
+          return JSON.parse(networkObject.curves).reduce((all2, curve) => {
+            all2.push({ HDPath, curve })
+            return all2
+          }, [])
+        },
+        []
+      )
+      const foundCombination = await Promise.all(
+        walletVariations.map(async ({ HDPath, curve }) => {
+          const wallet = await getWallet(
+            seedPhrase,
+            networkObject,
+            HDPath.value,
+            curve.value
+          )
+          return wallet && wallet.cosmosAddress === address ? true : false
+        })
+      )
+      return foundCombination.find((combination) => combination) ? true : false
```

Comment:
this can return `[null]`?
```suggestion
      return foundCombination.find((combination) => combination) ? true : false
```

I like to add `!!` to show that we are checking for existance
```suggestion
      return foundCombination.find((combination) => !!combination) ? true : false
```